Extensions/ShareTo/ShareViewController.swift

```diff
@@ -25,6 +25,38 @@ protocol ShareControllerDelegate {
     func shareController(shareController: ShareDialogController, didShareItem item: ShareItem, toDestinations destinations: NSSet) -> Void
 }
 
+private struct ShareDialogControllerUX {
+    static let CornerRadius: CGFloat = 4                                                            // Corner radius of the dialog
+
+    static let NavigationBarTintColor = UIColor(rgb: 0xf37c00)                                      // Tint color changes the text color in the navigation bar
+    static let NavigationBarCancelButtonFont = UIFont.systemFontOfSize(UIFont.buttonFontSize())     // System default
+    static let NavigationBarAddButtonFont = UIFont.boldSystemFontOfSize(UIFont.buttonFontSize())    // System default
+    static let NavigationBarIconSize = 32                                                           // Width and height of the icon
+    static let NavigationBarBottomPadding = 12
+
+    static let ItemTitleFont = UIFont(name: "HelveticaNeue-Medium", size: 12)
```

Comment:
We've made the decision to use Helvetica Neue everywhere, which is the same as the system font. However, we sometimes need variants like *Medium* or an odd point size for which there is no `UIFont` convenience method.

I agree that this is confusing and I am not entirely sure how to deal with this.